### PR TITLE
Dockerfile,contrib,hack: remove buildpack-deps

### PR DESCRIPTION
**Pull request by cpuguy83**
Carry #19504

### CODE REVIEW

**Comment by jfrazelle** [comment]
LGTM thanks! jankyyyyyyy be green

**Comment by calavera** [comment]
> jankyyyyyyy be green

:pray:

**Comment by calavera** [comment]
:cry: :sob:

File: hack/make/.ensure-syscall-test [hunk 1]
```diff
@@ -4,5 +4,20 @@ set -e
...
-	docker build ${DOCKER_BUILD_ARGS} -qt syscall-test contrib/syscall-test > /dev/null
+	if [ "$DOCKER_ENGINE_OSARCH" = "$DOCKER_CLIENT_OSARCH" ]; then
+		tmpdir=$(mktemp -d)
+		gcc -g -Wall -static contrib/syscall-test/userns.c -o "${tmpdir}/userns-test"
+		gcc -g -Wall -static contrib/syscall-test/ns.c -o "${tmpdir}/ns-test"
+		gcc -g -Wall -static contrib/syscall-test/acct.c -o "${tmpdir}/acct-test"
+
+		dockerfile="${tmpdir}/Dockerfile"
+		cat <<-EOF > "$dockerfile"
+		FROM debian:jessie
```
Comment (unclejack) [inline hack/make/.ensure-syscall-test]: I guess this is better than before, but it's still going to take a while.

Comment (cpuguy83) [inline hack/make/.ensure-syscall-test]: pretty fast, seccomp test takes 1/10th as long.

**Comment by cpuguy83** [comment]
Failures are unrelated, merging.

File: hack/make/.ensure-syscall-test (hunk 1 above)
Comment (unclejack) [inline hack/make/.ensure-syscall-test]: fair enough

**Comment by unclejack** [comment]
LGTM

**Pull request merged**